Fabulous.Core/ViewElement.fs

```diff
@@ -45,6 +46,9 @@ type ViewElement internal (targetType: Type, create: (unit -> obj), update: (Vie
     static member Create (create: (unit -> 'T), update: (ViewElement voption -> ViewElement -> 'T -> unit), attribsBuilder: AttributesBuilder) =
         ViewElement(typeof<'T>, (create >> box), (fun prev curr target -> update prev curr (unbox target)), attribsBuilder.Close())
 
+    [<System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)>]
+    static member val _CreatedAttribKey : AttributeKey<obj -> unit> = AttributeKey<_>("ElementCreated")
```

Comment:
Is it feasible to expose the real type through `created: ('T -> unit)`?
This would help avoid bad conversion, and expose the hidden custom types (CustomListView).